src/main/java/musical/squares/SquareMouseListener.java

```diff
@@ -16,8 +16,7 @@ public void mouseClicked(MouseEvent e) {
         int rowIndex = row / SquaresView.BORDERED_CELL_SIZE;
         int colIndex = col/ SquaresView.BORDERED_CELL_SIZE;
 
-        //TODO:Add toggle method that returns boolean value
-        squares.setCell(rowIndex, colIndex, !squares.getCell(rowIndex, colIndex));
+        squares.toggleCell(rowIndex, colIndex);
```

Comment:
I think this method should return the boolean value that the cell becomes and should be called in the if statement